packages/p2p/src/components/buy-sell/buy-sell-row.jsx

```diff
@@ -93,17 +93,25 @@ const BuySellRow = ({ row: advert }) => {
                         </div>
                         <div className='buy-sell-row__rating'>
                             {!!rating_count && !!rating_average ? (
-                                <StarRating
-                                    empty_star_className='buy-sell-row__rating--star'
-                                    empty_star_icon='IcEmptyStar'
-                                    full_star_className='buy-sell-row__rating--star'
-                                    full_star_icon='IcFullStar'
-                                    initial_value={rating_average_decimal}
-                                    is_readonly
-                                    number_of_stars={5}
-                                    should_allow_hover_effect={false}
-                                    star_size={14}
-                                />
+                                <div className='buy-sell-row__rating--row'>
+                                    <Text color='less-prominent' size={isMobile() ? 'xxxs' : 'xxs'}>
+                                        {rating_average_decimal}
+                                    </Text>
+                                    <StarRating
+                                        empty_star_className='buy-sell-row__rating--star'
+                                        empty_star_icon='IcEmptyStar'
+                                        full_star_className='buy-sell-row__rating--star'
+                                        full_star_icon='IcFullStar'
+                                        initial_value={rating_average_decimal}
+                                        is_readonly
+                                        number_of_stars={5}
+                                        should_allow_hover_effect={false}
+                                        star_size={14}
+                                    />
+                                    <Text color='less-prominent' size={isMobile() ? 'xxxs' : 'xxs'}>
+                                        ({rating_count})
```

Comment:
```suggestion
                                        {rating_count}
```

Reply:
i think this has to be left like this cause in design, the rating count should be `(100)` in between parenthesis